examples/nlp/md/multiple_choice_task_with_transfer_learning.md

```diff
@@ -692,71 +916,6877 @@ history = model.fit(
 <div class="k-default-codeblock">
 ```
 Epoch 1/5
- 183/183 ━━━━━━━━━━━━━━━━━━━━ 5087s 25s/step - accuracy: 0.2563 - loss: 1.3884 - val_accuracy: 0.5150 - val_loss: 1.3742 - learning_rate: 1.0000e-06
-Epoch 2/5
- 183/183 ━━━━━━━━━━━━━━━━━━━━ 4529s 25s/step - accuracy: 0.3825 - loss: 1.3364 - val_accuracy: 0.7125 - val_loss: 0.9071 - learning_rate: 2.9000e-06
-Epoch 3/5
- 183/183 ━━━━━━━━━━━━━━━━━━━━ 4524s 25s/step - accuracy: 0.6144 - loss: 1.0118 - val_accuracy: 0.7425 - val_loss: 0.8017 - learning_rate: 4.8000e-06
-Epoch 4/5
- 183/183 ━━━━━━━━━━━━━━━━━━━━ 4522s 25s/step - accuracy: 0.6744 - loss: 0.8460 - val_accuracy: 0.7625 - val_loss: 0.7323 - learning_rate: 4.7230e-06
-Epoch 5/5
- 183/183 ━━━━━━━━━━━━━━━━━━━━ 4517s 25s/step - accuracy: 0.7200 - loss: 0.7458 - val_accuracy: 0.7750 - val_loss: 0.7022 - learning_rate: 4.4984e-06
+
+WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
+I0000 00:00:1704605251.846942   12962 device_compiler.h:186] Compiled cluster using XLA!  This line is logged at most once for the lifetime of the process.
 
 ```
 </div>
----
-## Inference
+    
+  1/183 [..............................] - ETA: 16:51:51 - loss: 1.3865 - accuracy: 0.3750
 
+<div class="k-default-codeblock">
+```
+
+```
+</div>
+  2/183 [..............................] - ETA: 44:52 - loss: 1.3652 - accuracy: 0.3750   
 
-```python
-# Make predictions using the trained model on last validation data
-predictions = model.predict(
-    valid_ds,
-    batch_size=CFG.batch_size,  # max batch size = valid size
-    verbose=1,
-)
+<div class="k-default-codeblock">
+```
+
+```
+</div>
+  3/183 [..............................] - ETA: 44:29 - loss: 1.3851 - accuracy: 0.2500
 
-# Format predictions and true answers
-pred_answers = np.arange(4)[np.argsort(-predictions)][:, 0]
-true_answers = valid_df.label.values
+<div class="k-default-codeblock">
+```
+
+```
+</div>
+  4/183 [..............................] - ETA: 44:12 - loss: 1.3875 - accuracy: 0.2188
 
-# Check 5 Predictions
-print("# Predictions\n")
-for i in range(0, 50, 10):
-    row = valid_df.iloc[i]
-    question = row.startphrase
-    pred_answer = f"ending{pred_answers[i]}"
-    true_answer = f"ending{true_answers[i]}"
-    print(f"❓ Sentence {i+1}:\n{question}\n")
-    print(f"✅ True Ending: {true_answer}\n   >> {row[true_answer]}\n")
-    print(f"🤖 Predicted Ending: {pred_answer}\n   >> {row[pred_answer]}\n")
-    print("-" * 90, "\n")
+<div class="k-default-codeblock">
+```
+
+```
+</div>
+  5/183 [..............................] - ETA: 43:54 - loss: 1.3896 - accuracy: 0.2000
```

Comment:
That's still using Keras 2.

Make sure to install keras 3 *after* everything else is installed.

Reply:
> That's still using Keras 2.
> 
> Make sure to install keras 3 _after_ everything else is installed.

Hi @sachinprasadhs Any update on this PR? Please. Thank you!